charts/prometheus-blackbox-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Prometheus Blackbox Exporter
 name: prometheus-blackbox-exporter
-version: 7.9.0
+version: 7.9.1
```

```suggestion
version: 7.10.0
```

0.24 release [introduces](https://github.com/prometheus/blackbox_exporter/releases/tag/v0.24.0) some behavior changes, so I'd suggest to bump minor version instead.

---

There was a minor change from 7.8.0 to 7.9.0.
Within PR #3457 everything was updated to use v0.24.0, but somehow the image-tag was not updated.
You could see my change as a small bugfix as "patch release".

---

OK, I agree.
Thanks for clarifying this!

I will update my PR this afternoon, or feel free to fix it on your own if you like.